### PR TITLE
LPS-192618 Replace usages of aui:button in portlet-configuration-web

### PR DESCRIPTION
**Pull request by Stephy6**
Motivation
=======
The motivation behind this PR stadirize the usages of buttons in the portal by using for clay:button or clay:icon.
https://liferay.atlassian.net/browse/LPS-192618

Proposed Solution
========
The proposed solution consist in substitute the usages of aui:button for clay:button or clay:icon.

Steps to Verify
========
1. Go to the Page Editor and add an asset-publisher
2. Click on the Options dropdown button of the fragment.
3. Click on the Permissions button and check that the Save and Cancel buttons work fine.
4. From the dropdown again, now click over Configuration Templates. Check that the `Save current configuration as template` button works fine.

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/portlet-configuration/portlet-configuration-web/src/main/resources/META-INF/resources/edit_permissions.jsp [hunk 1]
```diff
@@ -219,9 +219,16 @@ if (Validator.isNotNull(portletConfigurationPermissionsDisplayContext.getModelRe
...
-		<aui:button name="saveButton" type="submit" />
+		<clay:button
+			id='<%= liferayPortletResponse.getNamespace() + "saveButton" %>'
+			label="save"
+			type="submit"
+		/>
 
-		<aui:button type="cancel" />
+		<clay:button
+			displayType="btn-secondary cancel"
```
Comment (victorg1991) [inline modules/apps/portlet-configuration/portlet-configuration-web/src/main/resources/META-INF/resources/edit_permissions.jsp:229]: This should be something like "secondary" only right?

Apart from that it seems that the class "btn-cancel" is needed for the button to work

**Review by victorg1991 — COMMENTED** [review]
Hey @Stephy6 The cancel button is not currently not working and the styles are a bit off, could you take a look, please? :)

File: modules/apps/portlet-configuration/portlet-configuration-web/src/main/resources/META-INF/resources/edit_permissions.jsp (hunk 1 above)
Comment (Stephy6) [reply modules/apps/portlet-configuration/portlet-configuration-web/src/main/resources/META-INF/resources/edit_permissions.jsp:229]: Yes, thats true

**Comment by Stephy6** [comment]
> Hey @Stephy6 The cancel button is not currently not working and the styles are a bit off, could you take a look, please? :)

Hey @victorg1991 , the sf cha

> Hey @Stephy6 The cancel button is not currently not working and the styles are a bit off, could you take a look, please? :)

Yes, I added `btn-cancel` to displayType and the sf changed it, now I get it. Thanx @victorg1991

**Comment by victorg1991** [comment]
ci:forward

**Comment by liulukiwi** [comment]
ci:test:relevant

**Comment by liulukiwi** [comment]
ci:stop

**Comment by liulukiwi** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/139719

**Comment by liulukiwi** [comment]
ci:close

**Pull request closed without merging**